### ISSUE TITLE
Accessibility issue : Elements that have scrollable content should be accessible by keyboard

### ISSUE THREAD

**Issue by tejamora** [microsoft/BotFramework-WebChat#3567]
Running a fast pass on WebChat bot, getting this error

Path | .react-scroll-to-bottom--css-qgskw-1n7m0yu

Fix ONE of the following:
Element should have focusable content
Element should be focusable

## Screenshots

![accessbility issue](https://user-images.githubusercontent.com/73717431/97748854-5acc4b80-1aab-11eb-985b-b6eede48b033.jpg)

![minimizechatbot](https://user-images.githubusercontent.com/73717431/97747553-60c12d00-1aa9-11eb-8fde-5dde8b6e7ad0.jpg)

Same replicable on Web Chat's minimizable sample: https://microsoft.github.io/BotFramework-WebChat/06.recomposing-ui/a.minimizable-web-chat/
![minimizesample](https://user-images.githubusercontent.com/73717431/97747554-63bc1d80-1aa9-11eb-9cd8-0ed219eec7dd.png)

## Version

 "botframework-webchat": 
      "version": "4.10.1",

## Steps to reproduce

1. Go to chatbot icon
1. Click on chatbot
1. when there are numerous activities between the bot and user

[Bug]

**Comment by CezaryMarcjan** [comment]
Hi @tejamora the team will review and follow up on this. Thanks for the report.

**Comment by corinagum** [comment]
This needs a team discussion, as right now the transcript intentionally only allows focus on interact-able elements.

**Comment by compulim** [comment]
A previous issue on "should not focus on non-interactive element" (#1340) and [WCAG 2.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-focus-order.html).

Since this accessibility issue is conflicting to one we previously fixed, we should raise this issue to internal accessibility team and ask them to clarify, and possibly a correct/reference implementation.

For instance, WhatsApp Web does not allow focus on the transcript. But pressing <kbd>PGUP</kbd>/<kbd>PGDN</kbd> keys will page the transcript up/down.

**Comment by corinagum** [comment]
For consolidation, I am closing this issue in favor of #3567. Sorry for the inconvenience.

**Comment by edemehmetcan** [comment]
Please fix this problem asap. The government in Europe is asking that the chatbot is fully compatible with WCAG

**Comment by corinagum** [comment]
Please note that this is available as of the 4.12.0 release.
